CONTRIBUTING.md

```diff
@@ -122,3 +122,18 @@ from the main (upstream) repository:
 [github]: https://github.com/gdg-x
 [pluspage]: https://google.com/+GDGXProject
 [js-style-guide]: http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml
+
+## <a name="members"></a> GDG[x] contributors
+Everybody is invite to dive into the code, explore, learn, add issues and send pull requests.
```

Comment:
Everybody is invited to dive into the code, explore, learn, open issues, and send pull requests.